simulator/simulator.py

```diff
@@ -88,7 +88,7 @@ def run_data_generation() -> None:
         data: EnvData = next(generator)
         insert_data(data)
         generated_data_amount += 1
-        time.sleep(1)
+        time.sleep(0.1)
```

Comment:
[nit]
It seems updating too fast for charts in my opinion.
Maybe we can try to decrease the number of coordinates to make it display better.

Reply:
Addressed in [544db48](https://github.com/ricky9667/smart_space/pull/22/commits/544db48f3d06d3a745570e602aa36abf2aaabe4b).